code/lib/cli-storybook/src/automigrate/fixes/addon-a11y-addon-test.test.ts

```diff
@@ -241,4 +558,133 @@ describe('addonA11yAddonTest', () => {
       `);
     });
   });
+
+  describe('transformPreviewFile', () => {
+    it('should add a new tags property if it does not exist', async () => {
+      const source = dedent`
+        import type { Preview } from '@storybook/react';
+
+        const preview: Preview = {
+          parameters: {
+            controls: {
+              matchers: {
+                color: /(background|color)$/i,
+                date: /Date$/i,
+              },
+            },
+          },
+        };
+
+        export default preview;
+      `;
+
+      const transformed = await transformPreviewFile(source, process.cwd());
+
+      expect(transformed).toMatchSnapshot();
```

Comment:
can we do `toMatchInlineSnapshot()` instead, or are they too big? This doesn't really show that it actually works, which is a shame.

Reply:
Good point! I will update it